roofit/roofit/src/RooGaussExpTails.cxx

```diff
@@ -0,0 +1,110 @@
+/** \class RooGaussExpTails
+    \ingroup Roofit
+
+PDF implementing a Gaussian core + double-sided exponential tail distribution
+\author Souvik Das (8/1/2013) Initial implementation and
+Giovanni Marchiori (30/3/2016) Implemented analytic integral
+\see http://arxiv.org/pdf/1603.08591v1.pdf, https://github.com/mpuccio/RooCustomPdfs/blob/master/RooGausDExp.cxx, https://doi.org/10.1142/S0217751X14300440
+\note To use the one-sided version, just set the opposite parameter k to a very large value
+*/
+
+
+#include "RooGaussExpTails.h"
+#include "RooAbsReal.h"
+#include <cmath>
+#include "Math/ProbFuncMathCore.h"
+
+ClassImp(RooGaussExpTails)
+
+
+//_____________________________________________________________________________
+RooGaussExpTails::RooGaussExpTails(const char *name, const char *title,
+                            RooAbsReal::Ref _x,
+                            RooAbsReal::Ref _x0,
+                            RooAbsReal::Ref _sigma,
+                            RooAbsReal::Ref _kL,
+                            RooAbsReal::Ref _kH) :
```

Comment:
```suggestion
                            RooAbsReal::Ref x,
                            RooAbsReal::Ref x0,
                            RooAbsReal::Ref sigma,
                            RooAbsReal::Ref kL,
                            RooAbsReal::Ref kH) :
```
The constructor arguments on the other hand should not have trailing or leading underscores, just like local viariables